addons/categories/widget.php

```diff
@@ -57,9 +71,6 @@ public function widget( $args, $instance ) {
 			'order'      => $instance['order'],
 		);
 
-		$icon_width  = ! empty( $instance['icon_width'] ) ? $instance['icon_width'] : 32;
```

Comment:
This two lines are still needed, check my comment on old PR.

Reply:
Also add another check of `is_numeric` to make sure received values are valid pixels in number.